### PR TITLE
Wip master

### PR DESCRIPTION
**Pull request by jordanhudgens**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,33 @@
...
 3. Draw routes with variables
+
 4. Generate URLs using route helpers that require variables
 
-## Notes
 
-Goal of this lab is to build an index and show action using students where index links to students show and each students show links back to the index.
+## Instructions
+
+Currently four tests are currently failing and need implementation code to start passing. In order to complete this lab you will need to complete the following tasks:
```
Comment (jmburges) [inline README.md]: I would re-word these to be more user story focused. What are the students actually building? The specifics of how that should be built will be in the tests, no need to explain it to them

File: README.md [hunk 2]
```diff
@@ -3,14 +3,33 @@
...
+
+* Update the ```config/routes.rb``` file and utilize the ```:as``` call for the ```show``` route to create the ```student_path(:id)``` helper method.
```
Comment (jmburges) [inline README.md]: Also when you give the order of operations, I try and go from easiest thing to hardest. In the same way that when you're playing games the levels go up increasing levels of difficulty

File: config/routes.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  resources :students, only: :index
+
+  # This show route can be refactored into the above resouces' method call above, like so:
+  # resources :students, only: [:index, :show]
+  # However for the sake of this lab and seeing
+  # how you can pass params through the
+  # route, we'll keep it explicit and refactor it later
+  get "students/:id", to: "students#show"
```
Comment (jmburges) [inline config/routes.rb:9]: I assume this is solution stuff?

File: spec/factories/students.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
```
Comment (jmburges) [inline spec/factories/students.rb]: No factories, just create the stuff normally

File: spec/features/student_spec.rb [hunk 5]
```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+feature 'Route to view' do
```
Comment (jmburges) [inline spec/features/student_spec.rb]: change to describe/it

**Comment by jordanhudgens** [comment]
@jmburges This has been updated with your requested changed, here is the [solution branch](https://github.com/learn-co-curriculum/rails-url-helpers-lab/tree/wip-solution)

**Comment by jmburges** [comment]
In the interest of time, I'm going to merge this in and fix it myself. The big issue with this lab is that the way you are doing the `as` is so unnatural. We are forcing them into something that I feel like I would never do. I'm going to use a different `as` so that it feels a bit more natural.

**Comment by jmburges** [comment]
also @jordanhudgens they don't know what the `before_action` stuff is

**Pull request merged**